Delicious/Utils/UI/EmptyView.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  EmptyView.swift
+//  Delicious
+//
+//  Created by HoaPQ on 7/31/20.
+//  Copyright © 2020 HoaPQ. All rights reserved.
+//
+
+import Foundation
+
+class EmptyView: UIView, NibLoadable {
```

Comment:
Suggest: cái này em hoàn toàn có thể cấu trúc tối ưu và clear hơn bằng cách sau:
- Khởi tạo và gàn backgroundView cho Tableview.
- Viết các binding property cho backroundView của tableView với điều kiện backgroundView đó có kiểu dữ liệu là `EmptyView`.
- Mỗi lần có error hay message gì thì chỉ cần binding vào là xong.
Còn như anh thấy cách hiện tại sử dung closure thì vẫn được, tuy nhiên sau thời gian dài thì code sẽ nở ra khá nhiều và khó maintaint